src/styles/blocks/calendar.scss

```
@@ -0,0 +1,24 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calcCalendarWidth($paddings, $calendarDaysSize, $gap);
+  gap: 1px;
+  padding: 10px;
+
+  &__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    @include square($calendarDaysSize);
+    background-color: #eee;
+    border: 1px solid #000;
+    transition-duration: 0.5s;
```

Always list styles that you apply transition for:
```suggestion
    transition: transform 0.5s, background-color 0.5s;
```